pkg/cloud/vsphere/provisioner/common/templates.go

```diff
@@ -250,6 +266,14 @@ users:
   sudo: ALL=(ALL) NOPASSWD:ALL
   groups: sudo
   shell: /bin/bash
+{{- if .TrustedCerts }}
```

Comment:
will this affect kubeadm?  looks like not, as i don't see certs been passed to kubeadm(e.x --cert-dir).
so this trusted cert is not mean for Kubernetes cluster, correct? thanks

Reply:
Well, for one cluster-api is the one responsible to deploy and setup the node. Also, I think the certs could potentially be used by kubernetes services as well (not saying it will be, but could be)

Reply:
@figo We did something similar in VIC to pull in certificates for private container registries.